README.md

```diff
@@ -33,6 +99,8 @@ let image = Image(color: .green, width: 256, height: 256)
 ```
 Doing this for the entire API will increase discoverability and make Raylib for Swift a more Swifty experience.
 
+---
+Below is the original [Raylib](https://github.com/raysan5/raylib) README.
```

Comment:
Make this a separated heading

---
# Below is the original [Raylib](https://github.com/raysan5/raylib) README.


and delete:
---
 # Official Readme